src/middlewares/checkOwnerToken.js

```diff
@@ -0,0 +1,20 @@
+import errorRes from '../helpers/successHandler';
+import jwt from 'jsonwebtoken';
+import { config } from 'dotenv';
+
+config();
+
+export default async (req, res, next) => {
+  try {
+    const token = req.headers.auth.split(' ')[1];
+    const info = jwt.decode(token, process.env.JWT_KEY);
+    const userId = info.id;
+    if (parseInt(req.params.userId) !== userId && req.user.role !== 'admin') {
```

Comment:
should the operator be `&` or `||`?  if we use `&` that means if one of the statements is not met, the whole function becomes `false` regardless of the other check. So I think we use the `or` operator instead. What do you think?